PWGHF/DataModel/DerivedTables.h

```diff
@@ -287,18 +293,12 @@ DECLARE_SOA_TABLE(HfD0Pars, "AOD", "HFD0PAR", //! Table with candidate propertie
                   hf_cand::ImpactParameter1,
                   hf_cand_par::ImpactParameterNormalised0,
                   hf_cand_par::ImpactParameterNormalised1,
-                  hf_cand_par::NSigTpcPi0,
-                  hf_cand_par::NSigTpcKa0,
-                  hf_cand_par::NSigTofPi0,
-                  hf_cand_par::NSigTofKa0,
-                  hf_cand_par::NSigTpcTofPi0,
-                  hf_cand_par::NSigTpcTofKa0,
-                  hf_cand_par::NSigTpcPi1,
-                  hf_cand_par::NSigTpcKa1,
-                  hf_cand_par::NSigTofPi1,
-                  hf_cand_par::NSigTofKa1,
-                  hf_cand_par::NSigTpcTofPi1,
-                  hf_cand_par::NSigTpcTofKa1,
+                  hf_cand_par::NSigTpcPi,
+                  hf_cand_par::NSigTpcKa,
+                  hf_cand_par::NSigTofPi,
+                  hf_cand_par::NSigTofKa,
+                  hf_cand_par::NSigTpcTofPi,
+                  hf_cand_par::NSigTpcTofKa,
```

Comment:
I think it is clearer and less prone to typos, if the order is grouped by the PID hypothesis, as it was by the prong number before:
```suggestion
                  hf_cand_par::NSigTpcPi,
                  hf_cand_par::NSigTofPi,
                  hf_cand_par::NSigTpcTofPi,
                  hf_cand_par::NSigTpcKa,
                  hf_cand_par::NSigTofKa,
                  hf_cand_par::NSigTpcTofKa,
```